src/runtime_src/core/tools/xbutil2/OO_HostMem.cpp

```diff
@@ -116,8 +116,9 @@ OO_HostMem::execute(const SubCmdOptions& _options) const
     }
     enable =  boost::iequals(m_action, "ENABLE");
 
-    // Exit if ENABLE action is specified but size is not
-    if(enable && size == 0)
+    // Exit if ENABLE action is specified and 
+    // size is not a power of 2
+    if(enable && !XBUtilities::isPowerOf2(size))
```

Comment:
I understand why you may have removed the "size==0" condition, since zero is not a power of two, but doing so does make the next developer work harder in understanding what you are testing for.   In addition, you are relying on a size effect of isPowerOf2() to determine a zero value.

This will come back an haunt the next developer if the power of 2 restriction is removed.

For this line only, would you please bring back the size comparison.
```suggestion
    if(enable && ((0 == size) || !XBUtilities::isPowerOf2(size)))
```